src/alarms/notifications/EditNotification.js

```diff
@@ -0,0 +1,69 @@
+import React, { Component } from "react";
+import { connect } from "react-redux";
+import { withRouter } from "react-router-dom";
+import MDSpinner from "react-md-spinner";
+
+import { NotificationForm } from "./NotificationForm";
+
+class EditNotificationModel extends Component {
```

Comment:
I'd like a comment saying what kind of props this component has (e.g. what format 'match' has). Or better, switch to Typescript and define it.

Reply:
Comment added